### ISSUE TITLE
Czech [cs]

### ISSUE THREAD

**Issue by Frooxius** [Frooxius/NeosLocale#1]
Coordination for the Czech locale. Initial translation was done during the conversion of Neos to the new localization system, but might need another pass after.

**Comment by rampa3** [comment]
I would like to help you with the Czech translation. Could I ask what strings are you working on now? I don't want to make collisions in the changes as stated in the readme. Actually I have translated the audio clip strings + other stuff not translated yet in a local copy. It is not on my GitHub fork yet... I had already validated the file and tested it in Neos (except for locomotion, because I am a desktop user).

**Comment by Frooxius** [comment]
I'm not currently working on any strings, feel free to translate them at all or even make changes to the ones I already did, since a bunch of them are probably weird (I always use software in English, so I'm not too familiar with common Czech terminology for some things). I'd definitely appreciate the help!

**Comment by rampa3** [comment]
I need an opinion: Should 3D model terminology be translated? I mean, at least from what I talked with people and seen/heared, we used original terminology without translating it to Czech...

**Comment by rampa3** [comment]
Update to the question given in last comment :
I asked around in my Czech Discord contacts people who do 3D modeling, and everyone said that the terminology should be in english, as that is the usual language for it. I will check Czech locale of Blender for reference, and if it has the terminology in english, I will go and revert all 3D modeling related terms to english, while leaving parts of the strings around them translated.

**Comment by Frooxius** [comment]
Yeah that sounds good! It will definitely help with interoperability with other tools that use the same terminology. Plus translating technical terms often tends to sound quite weird anyways.

**Comment by rampa3** [comment]
Ok, I have checked Blender as reference, and the results are that the key terms are more or less in english, just some are translated (skeleton ->kostra, mesh->síť), but I think that people who do modeling are familiar with the terms in their original form. I am going to revert all 3D modeling terms to english now...

**Comment by rampa3** [comment]
I am confused by the change in the MTC signs...
![obrazek](https://user-images.githubusercontent.com/68955305/99548003-1815d900-29b0-11eb-9242-985e7e0e9412.png)
 Like, I know what it means in US English, but I have no idea howe to translate it without making it sound weird or make no sense in Czech...

**Comment by rampa3** [comment]
Update: I have discussed with others in Discord and made a first guessed draft translation. If I got the meaning wrong, please inform me.

**Comment by rampa3** [comment]
What exactly is meant by "Homes in the exit strings:
- "Dash.Exit.ExitAndSave" : "Exit & Save Homes"
- "Dash.Exit.ExitAndDiscard" : "Exit & Discard Homes"
?

I feel like now used Czech versions of the strings are not describing exact meaning, but I am not sure if it means just the home world, or all changed worlds you own (as it sounds for me now).

**Comment by rampa3** [comment]
Also, any comments to this Grab World locomotion translation: "Zachytávání se světa - Vytvoří kotvu světa s referencí a přesouvá uživatele relativně k ní."? I don't know why, but I feel like it is kinda off, and not in proper Czech...

**Comment by Frooxius** [comment]
"Homes" in this case are the home worlds - the Local starting world and the Cloud home. It doesn't affect any other worlds that are open.

Grab World is probably a bit weird to translate. I wouldn't stress about that one too much though, since it's one of the legacy locomotions that's rarely used. It essentially lets you create a world-space anchor and move yourself relative to it - almost as if you grabbed the world itself and started moving it around with your controller if that makes sense.

**Comment by rampa3** [comment]
[cs_test.zip](https://github.com/Frooxius/NeosLocale/files/5785787/cs_test.zip)

Here is the outcome of pass 5 of proofreading of the Czech locale. I am not going to PR it immediately, since there was a considerable chunk of changes and I think I should get some feedback first (and test it myself). Posting it here in the test form, so you can take a look if you want.

**Comment by rampa3** [comment]
[cs-test(proof5-changes).txt](https://github.com/Frooxius/NeosLocale/files/5786079/cs-test.proof5-changes.txt)

Here is a summary for better orientation in the changes.

**Comment by rampa3** [comment]
I have just pushed a pull request for the big batch of fixes from the proofreading. I have found a little typo I made in the Steam store translation, so I fixed it in this batch too.

**Comment by rampa3** [comment]
I have a little problem with the newest batch of strings:
The translated string for key "Settings.AutosaveScreenshotPath" won't fit into the UI - it is too long, so it uses minuscule font when written in full length. This is as shortened as I was able to make it: "Cesta autom. ulož. screenshotů". Do you have any suggestions please?

**Comment by rampa3** [comment]
Got a little issue with the Neos bot creator:
![image](https://user-images.githubusercontent.com/68955305/112121114-c9bf5380-8bb6-11eb-9c1e-eadcbffe6db1.png)
Secondary colour in Czech is too long for the field, and I can't do the same thing as in English, where we just say "Secondary"...

**Comment by rampa3** [comment]
Sorry for delay - got HDD approaching EOL in my computer. It corrupts itself over time when in use, so I can't use the computer, if I want to have high chance on full clone without data loss. Replacement should arrive soon. When I replace that drive, I am going to do all the backlog ASAP!

**Comment by rampa3** [comment]
Just found a thing in the universe strings: assets =/= just models. You can say it like this too, but only if the assets you mean are ONLY models. Are there going to be listed only models in this section, or not? If not, I would suggest to just use the same I used already multiple times in normal Neos strings - "assety".

**Comment by rampa3** [comment]
Also I feel that this string might have misplaced line breaks and missing end of the parenthesis:
![image](https://user-images.githubusercontent.com/68955305/135726284-b5752679-7b4c-4bfd-bdff-d5f7ec8b6d27.png)
I am going to push fix with my newest translations.

**Comment by rampa3** [comment]
**IMPORTANT NOTICE:**
I have to resolve some pretty messy shenanigans with my study agenda right now. The updates of Czech locale might slow down even more, or stop completely until my study problems are resolved!
